indexer/src/main.ts

```diff
@@ -45,12 +53,6 @@ export const config = {
   finality: "DATA_STATUS_PENDING",
   filter: {
     header: { weak: false },
-    // Filters are unions
-    events: [
-      {
-        keys: [TRANSACTION_EXECUTED],
-      },
-    ],
```

Comment:
I think we can keep this and just use the `includeReverted` field from the filter. https://www.apibara.com/docs/networks/starknet/filter